### PR TITLE
Update centos repositories configuration

### PR DESCRIPTION
**Pull request by scollazo**
Fixes artefactual/archivematica#952

### CODE REVIEW

**Comment by scollazo** [comment]
Added fix for artefactual-labs/am-packbuild#38

**Review by sevein — APPROVED** [review]
Sweet!

File: admin-manual/installation/installation.rst [hunk 1]
```diff
@@ -444,7 +444,17 @@ Archivematica version 1.5.1 and higher support installation on CentOS/Redhat.
         [archivematica]
         name=archivematica
         baseurl=https://packages.archivematica.org/1.7.x/centos
-        gpgcheck=0
+        gpgcheck=1
+        baseurl=https://packages.archivematica.org/1.7.x/key.asc
```
Comment (sevein) [inline admin-manual/installation/installation.rst]: Here did you mean to use `gpgkey` instead?

**Review by sevein — CHANGES_REQUESTED** [review]
Actually I realize something seems wrong with your changes. This is the full snippet once your patch is applied:

```
sudo -u root bash -c 'cat << EOF > /etc/yum.repos.d/archivematica.repo
[archivematica]
name=archivematica
baseurl=https://packages.archivematica.org/1.7.x/centos
gpgcheck=1
baseurl=https://packages.archivematica.org/1.7.x/key.asc
enabled=1
EOF'

sudo -u root bash -c 'cat << EOF > /etc/yum.repos.d/archivematica-extras.repo
[archivematica-extras]
name=archivematica-extras
baseurl=https://packages.archivematica.org/1.7.x/centos-extras
gpgcheck=1
gpgkey=https://packages.archivematica.org/1.7.x/key.asc
enabled=1
EOF'
```

File: admin-manual/installation/installation.rst (hunk 1 above)
Comment (scollazo) [reply admin-manual/installation/installation.rst]: good catch!

**Review by sevein — APPROVED** [review]

**Comment by scollazo** [comment]
Merged as  d7c35fa

**Pull request closed without merging**